src/olympia/constants/webext_permissions.py

```diff
@@ -16,6 +16,7 @@
     u'http://*/*': ALL_URLS_PERMISSION,
     u'https://*/*': ALL_URLS_PERMISSION,
     u'*://*/*': ALL_URLS_PERMISSION,
+    # FIXME: how about https://*/path ?
```

Comment:
Yeah. I relied on your permission stuff to get me going but I'll need something else more robust in the long run, maybe in the linter (which could then copy/paste Firefox's logic entirely).

In this case, my concern is developers trying to game the system by submitting things like:
```json
permissions: [
    "*://*/a*",
    "*://*/b*",
    "*://*/c*",
    "*://*/d*",
   ...
]
```